api/comments.rb

```diff
@@ -55,6 +55,18 @@
       updated_content["endorsement"] = new_endorsed_val ? endorsement : nil
     end
   end
+  if params[:user_id]
+    if updated_content.has_key? BODY and updated_content[BODY] != comment.body
+      edit_reason_code = params.fetch("edit_reason_code", nil)
+      comment.edit_history.build(
+        original_body: comment.body,
+        author: user,
+        reason_code: edit_reason_code,
+        editor_username: user.username,
+      )
+      comment.save
```

Comment:
Nit: same comment here, I think we can remove this line since `save` gets invoked by `update_attributes`.